tests/Feature/Api/MediaTest.php

```diff
@@ -135,31 +178,26 @@ public function test_upload_media_not_allowed_model_type()
         $response->assertStatus(422);
     }
 
-    public function test_upload_media_file_already_exists()
+    public function test_upload_media_task_not_found()
     {
         $media = [
             'model_type' => $this->task->getMorphClass(),
-            'model_id' => $this->task->id,
+            'model_id' => ++$this->task->id,
             'media' => $this->file,
-            'is_public' => false,
         ];
 
         $this->user->givePermissionTo($this->permissions['upload']);
         Sanctum::actingAs($this->user, ['user']);
 
         $response = $this->actingAs($this->user)->post('/api/media', $media);
-        $response->assertStatus(201);
-
-        $reUpload = $this->actingAs($this->user)->post('/api/media', $media);
-        $reUpload->assertStatus(422);
+        $response->assertStatus(422);
     }
 
-    public function test_upload_media_task_not_found()
+    public function test_upload_media_media_field_missing()
```

Comment:
isn't this the same as test_upload_media_validation_fails()? Or what does it actually test for?